wallet/db.c

```diff
@@ -110,6 +110,17 @@ char *dbmigrations[] = {
     "  PRIMARY KEY (id),"
     "  UNIQUE (channel_id, channel_htlc_id, direction)"
     ");",
+    "CREATE TABLE invoices ("
+    "  id INTEGER,"
+    "  state INTEGER,"
```

Comment:
paid BOOLEAN  perhaps?

Reply:
sqlite3 emulates booleans with integers anyway, so I opted to make it explicit and maybe we want to add more states later on (abandoned, ...)